ukrainian/eat-wall/obese/continue/continue_eat.md

```diff
@@ -1,6 +1,5 @@
-��� �� ��� �� �����������, ��������� � ��i�i ���������� �i��, ����� ��� 
-������ ������i�� ������� ����i���.
-��������� ���� ���� �� �������� ������ ���, �� ����� ������, ��� ��� ����� ���� �� ������
-��������, ��� � ���� ��������. ����� ��� "������������" ���.
+Так як їли ви наполегливо, незабаром у стiнi утворилася дiра, через яку почало мерехтiти сонячне промiння. Пройшовши крізь стіну ви побачили дивний світ, де кожне дерево, кущ або квітка були не просто їстівними, але й дуже смачними. Такий собі "смаколиковий" сад.
```

Comment:
This line is longer than 120 characters. Could you break it up into multiple lines? That way it will be easier to read the raw files or view diffs on the command line.  No need to add blank lines in between, just press enter somewhere in the middle of the line.